src/vuestic-theme/vuestic-components/va-table/scenarios/VaTableSelect.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <va-card :title="$t('tables.selectable')">
+      <va-table
+        :fields="fields"
+        :data="data"
+        :per-page="5"
+      >
+        <template slot="select" slot-scope="props">
+          <va-checkbox :value="props.rowData.checked" @input="select(props.rowData)" />
+        </template>
+
+        <template slot="graph" slot-scope="props">
+          <svg width="100" height="40" xmlns="http://www.w3.org/2000/svg">
+            <path :d="props.rowData.graph" :stroke="props.rowData.graphColor" fill="transparent"/>
+          </svg>
+        </template>
+      </va-table>
+    </va-card>
+
+    <small>
+      {{ $t('tables.selected') }}: {{ selected }}
+    </small>
+  </div>
+</template>
+
+<script>
+import users from './users.json'
+
+export default {
+  data () {
+    return {
+      users: users,
+    }
+  },
+  computed: {
+    fields () {
+      return [{
+        name: '__slot:select',
+      }, {
+        name: 'fullName',
+        title: this.$t('tables.headings.name'),
+        width: '20%',
+      }, {
+        name: 'email',
+        title: this.$t('tables.headings.email'),
+        width: '30%',
+      }, {
+        name: 'country',
+        title: this.$t('tables.headings.country'),
+        width: '30%',
+      }, {
+        name: '__slot:graph',
+        dataClass: 'text-right',
+      }]
+    },
+    data () {
+      return this.users.map(user => {
```

Comment:
You likely want to use destructuring here. Or maybe just define `fullName` in data structure.

Makes no sense to use `map`, then modify and return item

Reply:
This was more for the demonstration purposes.

Reply:
Please suggest if it's better to ignore this and just add fullName to the data structure.

Reply:
@jean-moldovan I think keeping fullName in data structure is better solution, but not a huge difference.